.github/workflows/ci.yml

```diff
@@ -108,6 +108,8 @@ jobs:
           cargo install toml-cli
           ./.github/workflows/scripts/verify_tag.sh ${{ github.ref_name }} fuel-client/Cargo.toml
           ./.github/workflows/scripts/verify_tag.sh ${{ github.ref_name }} fuel-core/Cargo.toml
+          ./.github/workflows/scripts/verify_tag.sh ${{ github.ref_name }} txpool/Cargo.toml
+          ./.github/workflows/scripts/verify_tag.sh ${{ github.ref_name }} interfaces/Cargo.toml
```

Comment:
Nit, but can you sort these? I'll help future us not deal with an unsorted mess 😂